cmd/root.go

```diff
@@ -102,6 +107,34 @@ func logLevel() error {
 	return nil
 }
 
+func setupSentryHook() error {
+	sentryEnv := viper.GetString("sentry.env")
+	sentryDSN := viper.GetString("sentry.dsn")
+
+	sentryErr := sentry.Init(sentry.ClientOptions{
+		Dsn:         sentryDSN,
+		Environment: sentryEnv,
+		Debug:       true,
```

Comment:
Assuming I don't want to leave this as set to true as we merge to prod? Should it be made configurable?

Reply:
Definitely disable this in prod. I don't have a strong opinion on exposing this as a config option—I'd probably be OK removing it altogether.